app/views/report_mailer/report.text.erb

```diff
@@ -0,0 +1 @@
+The report is attached as a gzipped file.
```

Comment:
It would be good to inform the user about more things, such as what is the report name, who and when triggered that, perhaps even from which Foreman you receive it (setting -> foreman url).